src/main/java/io/github/xcube16/iseedragons/FakeThirstStatHandler.java

```diff
@@ -0,0 +1,128 @@
+package io.github.xcube16.iseedragons;
+
+import java.lang.reflect.Method;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.util.text.TextComponentString;
+import net.minecraft.world.World;
+import net.minecraftforge.event.entity.living.LivingEvent.LivingJumpEvent;
+import net.minecraftforge.event.entity.living.LivingHurtEvent;
+import net.minecraftforge.event.entity.player.AttackEntityEvent;
+import net.minecraftforge.event.world.BlockEvent;
+import net.minecraftforge.fml.common.FMLCommonHandler;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+public class FakeThirstStatHandler
+{
+	Object handler;
+	boolean initialized = false;
+	Method onPlayerJump;
+	Method onPlayerHurt;
+	Method onBlockBreak;
+	
+	Method getThirstData;
+	
+	Class thirstHandlerClass;
+	Method addExhaustion;
+	
+	public FakeThirstStatHandler(Object handler)
+	{
+		this.handler = handler;
+		try
+		{
+			onPlayerJump = handler.getClass().getMethod("onPlayerJump", LivingJumpEvent.class);
+			onPlayerHurt = handler.getClass().getMethod("onPlayerHurt", LivingHurtEvent.class);
+			onBlockBreak = handler.getClass().getMethod("onBlockBreak", BlockEvent.BreakEvent.class);
+			
+			getThirstData = Class.forName("toughasnails.api.thirst.ThirstHelper").getMethod("getThirstData", EntityPlayer.class);
+			
+			thirstHandlerClass = Class.forName("toughasnails.thirst.ThirstHandler");
+			addExhaustion = thirstHandlerClass.getMethod("addExhaustion", float.class);
+			
+			//No exceptions, report as successfully initialized
+			initialized = true;
+		}
+		catch(Exception e)
+		{
+			ISeeDragons.logger.error("Failed to initialize FakeThirstStatHandler");
+		}
```

Comment:
I don't feel too good about returning a FakeThirstStatHandler object when its <init> has failed (it will be filled with nulls n stuff). Maybe just throw the exception and handle it in ISeeDragons#postInit()?